### DISCUSSION TITLE
transcribe same audio but got different result

### DISCUSSION THREAD

**Discussion by genausz** [openai/whisper#1519] [General]
When i use the pretrained model (small) and infer the same audio file, but got different result. Anyone meet the same problem as me?

**Comment by glangford** [comment]
See in particular
https://github.com/openai/whisper/discussions/308#discussioncomment-3899382

and the following discussions:

* https://github.com/openai/whisper/discussions/81
* https://github.com/openai/whisper/discussions/274
* https://github.com/openai/whisper/discussions/308
